components/Navigation/DesktopSubnav/DesktopSubnav.tsx

```diff
@@ -102,7 +102,11 @@ export const DesktopSubnav = ({
                             </a>
                         </Link>
                         {link.title === 'Jobs' && jobsAvailable > 0 && (
-                            <Link href={urls.jobs} scroll={false}>
+                            <Link
+                                href={urls.jobs}
+                                scroll={false}
+                                legacyBehavior
+                            >
```

Comment:
As you mentioned it would be good to get a ticket to update these Links. If the a link is removed, can the other attributes be moved to the Link element?